app/src/main/java/com/ustwo/boilerplate/base/BasePresenter.java

```diff
@@ -7,7 +7,6 @@
 
 public class BasePresenter<T extends BaseView> {
     private final CompositeDisposable attachedDisposables = new CompositeDisposable();
-    private final CompositeDisposable visibleDisposables = new CompositeDisposable();
     private T view;
 
     protected BasePresenter() {
```

Comment:
Do we need a comment that BasePresenter is designed for extension? What if we make it abstract? If not, should we add unit tests to it?